.gitignore

```diff
@@ -8,4 +8,6 @@ node_modules/**
 build/**
 coverage/**
 packages/**
+umberto-output/**
```

Comment:
Yes, I will change default output path to be `build/docs/`. Also any project can change the output path by creating a custom config file for hexo.